tests/App/Jobs/FetchCollectionVolumeTest.php

```diff
@@ -40,3 +40,9 @@
 
     expect((new FetchCollectionVolume($collection))->retryUntil())->toBeInstanceOf(DateTime::class);
 });
+
+it('has a unique ID', function () {
+    $collection = Collection::factory()->create();
+
+    expect((new FetchCollectionVolume($collection))->uniqueId())->toBeString();
+});
```

Comment:
Don't you think this test need to be refined as we already have return type of the `uniqueId` method:

```PHP
public function uniqueId(): string
```

Reply:
don't think it brings any value. this test is purely to satisfy 100% coverage. should changing the implementation of a `uniqueId` to something else make the tests fail, even if the updated value ensures unique jobs? the point is to test what it does (uniqueness), not the actual implementation that might change

Reply:
1. the point is to test what it does (uniqueness), not the actual implementation that might change
2.  increasing the minutes from 10 to 15 is not crucial enough that should fail the tests

I do agree both of these statements, however, changing `uniqueId` implementation to something `hello-world` should make the test fail, I think. Even making it unique also should make the test fail just to warn  dev that you might mess up stuff, be careful and ensure that test is fixed. What do you think?